lcls-twincat-optics/lcls_twincat_optics_plc/POUs/FB_RMSWatch.TcPOU

```diff
@@ -0,0 +1,82 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_RMSWatch" Id="{7d0ac4a7-0c5c-406a-a694-5de5f07477d4}" SpecialFunc="None">
```

Comment:
I'd like to see something like this FB make its way into the motion lib at point